README.md

```diff
@@ -1,3 +1,114 @@
+- *Flexy Base Unit*: [https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28](https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28 "https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28")
+- *Others*: [https://websupport.ewon.biz/support](https://websupport.ewon.biz/support "https://websupport.ewon.biz/support")
+
+**Step 2:** Using a web browser, navigate to your Ewon's web page and select the `Users` tab, located under `Setup`.
+
+**Step 3:** Click `Add`, then create a username and password of your choice. Take note of this username and password, as it will be required in later setup steps. If you'd like, you may populate the `First Name`, `Last Name` and/or `Information` fields. IMPORTANT: Do not modify the rights of the new user.
+
+**Step 4:** Click `Add User` to complete the addition of the new user account.
+
+## Ignition Setup ##
+*Information for setup of Ignition (steps 1 - 4) can also be found here: [https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading](https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading "https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading")*
+
+**Step 1:** Using a web browser, download **Ignition 8** from [https://inductiveautomation.com/downloads/ignition](https://inductiveautomation.com/downloads/ignition "https://inductiveautomation.com/downloads/ignition").
+
+**Step 2:** After the download is complete, run the Ignition installer. The default installation options are satisfactory, although you may change them to fit your preference(s).
+
+**Step 3:** After Ignition is installed, the Ignition Gateway web page will open and guide you through commissioning Ignition. If the Ignition Gateway web page does not automatically open, using your browser, navigate to [http://localhost:8088](http://localhost:8088 "http://localhost:8088"). Take note of the username and password you set during commissioning, as this will be required in later setup steps.
+
+**Step 4:** After Ignition is commissioned, you will be redirected to the Ignition Gateway homepage. Take note of the URL in your browser's address bar as this will be required in later setup steps.
+
+**Step 5:** Download the HMS Ewon Connector from the EwonConnector releases page, found here: [https://github.com/hms-networks/EwonConnector/releases](https://github.com/hms-networks/EwonConnector/releases "https://github.com/hms-networks/EwonConnector/releases"). To download, expand the `Assets` menu under the latest Ignition 8 release, and click on the first file, commonly named `EwonConnection-X.Y.Z.zip`, where X.Y.Z is the version number.
```

Comment:
Maybe say they should be grabbing the latest release and the minimum supported version is 1.1.7